drivers/espi/espi_realtek_rts5912.c

```diff
@@ -2293,7 +2270,18 @@ static void espi_bus_reset_setup(const struct device *dev)
 		    DEVICE_DT_GET(DT_DRV_INST(0)), 0);
 	irq_enable(DT_IRQ_BY_NAME(DT_DRV_INST(0), bus_rst, irq));
 }
+#ifdef CONFIG_PM
+void espi_cs_low_isr(const struct device *port, struct gpio_callback *cb, gpio_port_pins_t pins)
+{
+	gpio_flags_t cs_pin_config;
 
+	gpio_pin_get_config(port, pins, &cs_pin_config);
+	if (cs_pin_config & GPIO_GCR_INTEN_Msk) {
```

Comment:
GPIO_GCR_INTEN_Msk is an internal bit for the GPIO controller.  You should check standard flags defined by gpio.h and gpio_defines.h

```suggestion
	if (cs_pin_config & GPIO_INT_ENABLE) {
```